devops/vaults.env

```diff
@@ -16,8 +16,8 @@ VUE_APP_AUTH_API_URL="op://API/$APP_ENV/auth-api/AUTH_API_URL"
 VUE_APP_AUTH_API_VERSION="op://API/$APP_ENV/auth-api/AUTH_API_VERSION"
 VUE_APP_LEGAL_API_URL="op://API/$APP_ENV/legal-api-le/LEGAL_API_URL"
 VUE_APP_LEGAL_API_VERSION_2="op://API/$APP_ENV/legal-api-le/LEGAL_API_VERSION_2"
-VUE_APP_NAICS_API_URL="op://API/$APP_ENV/naics-api/NAICS_API_URL"
-VUE_APP_NAICS_API_VERSION_2="op://API/$APP_ENV/naics-api/NAICS_API_VERSION"
+VUE_APP_NAICS_API_URL="op://API/$APP_ENV/naics-api-le/NAICS_API_URL"
```

Comment:
Has this been added to the 1Pass? I remember I was talking to Argus last week about it but it's not been there yet.

Reply:
Yes. I added it.

(Review the other PR first -- that's where I left some comments.)

Reply:
Awesome, thanks Sev!